hawkular-wildfly-agent/src/test/resources/org/hawkular/agent/monitor/extension/subsystem.xml

```diff
@@ -27,18 +27,20 @@
            availDispatcherBufferSize="101"
            availDispatcherMaxBatchSize="26">
 
+  <!-- default value omitted: reportTo="LOG" -->
   <diagnostics enabled="false"
-               reportTo="LOG"
                interval="1"
                timeUnits="minutes"/>
 
-  <storage-adapter type="HAWKULAR"
-                   username="jdoe"
+  <!-- default values omitted:
+          type="HAWKULAR"
+          inventoryContext="/hawkular/inventory/"
```

Comment:
we shouldn't even have to mention inventoryContext or metricsContext even in comments - these should never change and no one should really care these attributes exist. These are here only for the future in case we add the ability for the user to be able to change kettle's context from /hawkular to something else. So take those two out even in the comments.